Makefile

```diff
@@ -10,8 +10,8 @@ testjs: ## Clean and Make js tests
 testpy: ## Clean and Make unit tests
 	${PYTHON} -m pytest -v nbcelltests/tests --cov=nbcelltests
 
-testpy-forked: ## Python unit tests --forked (not windows!)
-	${PYTHON} -m pytest -v --forked nbcelltests/tests
+testpy-forked: ## Python unit tests parallelized across two CPUs
+	${PYTHON} -m pytest -v -n 2 --dist=loadscope nbcelltests/tests
```

Comment:
why `dist=loadscope`?

* it's different from the invocation on azure (ideally azure would be using the makefile targets, but that's another story...)
* are we having to rely on some ordering or grouping for the distributed case?

(i.e. I would prefer no guarantees about ordering/grouping - I'd like the distributed case to be least like the regular case)

Reply:
- I do not think it is different from the invocation on Azure. I think it just calls`make testpy-distributed` now which will come to Makefile and run the command with `--dist=loadscope`. Does it not?
- Those 5 tests need to be sequential.

Reply:
Please add a comment here explaining why loadscope (and ideally would link to a github issue).